lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -80,7 +91,7 @@ def install!
           # @return [Pathname] The absolute path of the Info.plist to use for an app host.
           #
           def app_host_info_plist_path
-            project.path.dirname.+("#{name}/#{name}-Info.plist")
+            project.path.dirname.+(group.path.to_s).+("#{name}/#{name}-Info.plist")
```

Comment:
this also looks weird -- I think if we use the group path, we shouldn't also need the `name` intermediary directory?